db/write_batch.cc

```diff
@@ -1967,6 +1975,34 @@ Status WriteBatchInternal::InsertInto(
   return s;
 }
 
+void WriteBatchInternal::AsyncInsertInto(
+    WriteThread::Writer* writer, SequenceNumber sequence,
+    ColumnFamilySet* version_set, FlushScheduler* flush_scheduler,
+    bool ignore_missing_column_families,
+    DB* db, SafeQueue<std::function<void()>>* pool) {
+  auto write_group = writer->write_group;
+  write_group->running.fetch_add(writer->batches.size(),
+                                 std::memory_order_relaxed);
```

Comment:
let's use seq_cst order consistently for `write_group->running` to make the code easier to read.

Reply:
ok